test/blackbox-tests/test-cases/interfere-sandbox-deletion.t

```diff
@@ -0,0 +1,33 @@
+Create a bad rule that doesn't allow the sandbox to be deleted.
+
+Note that this rule also fails. We make sure that we see both the error from
+the rule and the sandbox cleanup.
+
+  $ cat >dune <<EOF
+  > (rule
+  >  (target foo)
+  >  (action
+  >   (with-accepted-exit-codes 1
```

Comment:
Why? Was this command supposed to fail too? (It doesn't appear to, at the moment.)

Reply:
Yes, the intention is for the command to produce the target _and_ fail. It also does fail, see `Command exited with code 0.` in the error message.

Reply:
Oh, I see. Why not `exit 1` instead? Seems way simpler.

Reply:
(And maybe `echo` something to confirm the error gets captured and displayed.)

Reply:
Should be good now.